### PR TITLE
Feature android sfx

### PR DESCRIPTION
**Pull request by ThatFlint**
Sound effects play whenever a player gets hit, performs an attack, and succesfully hits an enemy.

### CODE REVIEW

**Comment by dVoorhout** [comment]
I think most of the audio stuff can be in its own class. The mainActivity is becoming a god class much like the main environment.

File: ContextApp/src/main/java/com/github/migi_1/ContextApp/MainActivity.java [hunk 1]
```diff
@@ -232,7 +256,49 @@ private PlatformPosition getRandomPosition() {
...
+    public void load() {
+        unload();
+        
+        soundIds[0] = soundPool.load(this, R.raw.gethit, 1);
+        soundIds[1] = soundPool.load(this, R.raw.miss, 1);
+        soundIds[2] = soundPool.load(this, R.raw.hit, 1);
+    }
+    
+    /**
+     * Setter for cooldown
```
Comment (marcelkuipers) [inline ContextApp/src/main/java/com/github/migi_1/ContextApp/MainActivity.java]: First sentence should end with a period

File: ContextApp/src/main/java/com/github/migi_1/ContextApp/MainActivity.java [hunk 2]
```diff
@@ -120,7 +131,20 @@ public void onResume() {
     @Override
     protected void onStop() {
         super.onStop();
-    }
```
Comment (marcelkuipers) [inline ContextApp/src/main/java/com/github/migi_1/ContextApp/MainActivity.java:123]: This method can be deleted now.

**Comment by ThatFlint** [comment]
The sound effects are now being played from the new SfxPlayer class.

**Comment by dVoorhout** [comment]
code looks a lot better, and all sounds seem to be working

**Comment by dVoorhout** [comment]
i approve

**Comment by marcelkuipers** [comment]
Looks good. Merging

**Pull request merged**